docs/source/transactions.md

```diff
@@ -2093,3 +2094,40 @@ A new Agreement needs to be sent instead.
 }
 ```
 
+#### LEDGERS_FREEZE
+
+Disable specific legders. System completely ignore frozen ledgers. After freeze you can delete frozen ledger via [remove_ledger.py] (https://github.com/hyperledger/indy-node/blob/master/scripts/remove_ledger.py).
```

Comment:
Could you change "disable" for a more suitable word and add more information about impact this txn to the system?